app/serializers/answer_serializer.rb

```diff
@@ -1,4 +1,9 @@
 class AnswerSerializer < ActiveModel::Serializer
+  include ActionView::Helpers::DateHelper
   attributes :id, :response, :accepted, :cached_votes_up, :cached_votes_down, :cached_votes_score, :created_at, :user, :comments
   
+  def created_at
+    time_ago_in_words(object.created_at)
```

Comment:
LOL. Is there actually a built in method called `time_ago_in_words`? #ohruby

Reply:
HAHAH Right?!